minestudio/utils/__init__.py

```
@@ -5,4 +5,17 @@
 FilePath: /MineStudio/minestudio/utils/__init__.py
 '''
 from .register import Register, Registers
-from .temp import get_mine_studio_dir
\ No newline at end of file
+from .temp import get_mine_studio_dir
```

Ruff (0.11.9)

`.temp.get_mine_studio_dir` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)